### ISSUE TITLE
Evilginx 3.3 - gophish integration - no reporting

### ISSUE THREAD

**Issue by matejsmycka** [kgretzky/evilginx2#1042]
Issue: Evilnginx is not reporting user actions (ReportEmailOpened...) to gophish.

Context:
https://breakdev.org/evilginx-3-3-go-phish/

**Comment by Ethansi2947E** [comment]
yo bro please can you guide me on how to setup mine with evilginx cus ive not had any experience with go phish

**Comment by Ethansi2947E** [comment]
do i need a new vps to host gophish?, how will the evilginx link be protected form emailscanners if the brae lure link is put there just like that?

**Comment by matejsmycka** [comment]
I figured it out. You have to use `{{.URL}}` parameter in the email template.

Otherwise correct GET args will not be there. So dont hardcode url links into email template.

This should be in documentation.

**Comment by Ethansi2947E** [comment]
> I figured it out. You have to use `{{.URL}}` parameter in the email template.
> 
> Otherwise correct GET args will not be there. So dont hardcode url links into email template.
> 
> This should be in documentation.

please can you teach me how to set it up cus im kinda lost

**Comment by Ethansi2947E** [comment]
how do i edit the template in  a way that the names and emails will be auromatically filled in the email templmate

**Comment by kpomeroy1979** [comment]
Use EvilGoPhish which is a perfect marriage of evilginx3 and gophish although the lure "redirect_url" functionality is broken, resulting in loops after users authenticate and go through MFA using a ms365 phishlet. Other then that, its great.

**Comment by matejsmycka** [comment]
However, evilgophish has its own use cases and advantages. 
There might be a good reason to use it, I am not that familiar with it.

What I meant is that there is no need for other integrations since there is an official one.

**Comment by Ethansi2947E** [comment]
> However, evilgophish has its own use cases and advantages. There might be a good reason to use it, I am not that familiar with it.
> 
> What I meant is that there is no need for other integrations since there is an official one.

so please how do i set it up tp work with my lure please

**Comment by Ethansi2947E** [comment]
that was the question ive been asking you

**Comment by Ethansi2947E** [comment]
i dont have good experirnce with gophish

**Comment by fin3ss3g0d** [comment]
I would honestly take a look at the project before making these kinds of misinformed statements, you are spreading misinformation to other people who might be reading this and may have a use case for the project. Just saying. You sound very biased for somebody who is unfamiliar with the project.

**Comment by matejsmycka** [comment]
Yes, I wrote it misleadingly, so I deleted the comments.

What I meant to say, evilnginx supports integration with gophish.

**Comment by ttinkywinky45** [comment]
**Hi 👽 ! If you are interested in ⚙️Advanced Phishing⚙️ and the ✨Evilginx3✨ tool then I offer you Google Gmail and Custom Phishlets with a ✅Free Tutorial✅ which arrives if the 📈Telegram reaches 50 Subscribers📈
⏳ GOOGLE GMAIL PHISHLET 2024 WITH PROOF ⏳
 ->
⛔https://www.youtube.com/watch?v=nA47UknIE6A⛔
---
🔓 Ask your questions and a free tutorial from 50 subs on telegram !!!
->
📩https://t.me/Noremorse1011📩**

![evilginx_background](https://github.com/kgretzky/evilginx2/assets/144027798/0e448f68-ecdc-49a4-a4f4-23dbed7cb971)

**Comment by fin3ss3g0d** [comment]
> Use EvilGoPhish which is a perfect marriage of evilginx3 and gophish although the lure "redirect_url" functionality is broken, resulting in loops after users authenticate and go through MFA using a ms365 phishlet. Other then that, its great.

Don't forget to mention `redirect_url` not working with o365 phishlet in EvilGophish is due to broken Evilginx code. The redirect code is taken directly from Evilginx, so Evilginx is also impacted by the issue.

**Comment by malikbens** [comment]
> I figured it out. You have to use `{{.URL}}` parameter in the email template.
> 
> Otherwise correct GET args will not be there. So dont hardcode url links into email template.
> 
> This should be in documentation.

I have the same problem but even if i use {{.URL}}, openned emails are never tracked .... i don't know if i am missing something.
When i investigate the email i see the html tag "<img>" whith a correct lure "https://domain/lure?l=afafoafeobeio" but no tracking in gophish

**Comment by malikbens** [comment]
Also if i manualy go the img src (wich is a evilginx lure ) its redirecting to the login page setup in the phishlets,  evilginx logged it as a new user landing ... i donct rly know if its intented to work like this or my installtion is not good

**Comment by matejsmycka** [comment]
yeah, i dont remeber now, but I had to modify source code to get it working.

**Comment by matejsmycka** [comment]
You literally have to be good with golang and fix a lot of stuff yourself.

Source code is half-baked, messy and PRs are not accepted….

**Comment by mahaabid2001** [comment]
I followed the steps mentioned here  https://breakdev.org/evilginx-3-3-go-phish/ , and the configuration was successfully done
![image](https://github.com/user-attachments/assets/74048c94-b453-456d-9bd5-621cfbebd805)
But I don't see the evilginx lure URL box displayed when creating a campaign on gophish. Is that normal?

**Comment by Ethansi2947E** [comment]
No it isn’t

**Comment by mahaabid2001** [comment]
is there a way to fix it?  or something I need to verify?  because I even
tried to uninstall and install gophish again and it is still not working.

Le ven. 9 août 2024 à 05:27, Ethansi2947E ***@***.***> a
écrit :

**Comment by mahaabid2001** [comment]
this is the config.json file for evilginx if that could help. Thank you.
[image: image.png]

**Comment by Hosted-EU** [comment]
Gmail phishlet available 2025✅ https://youtube.com/@ginxforcee?si=pJtePf5bbzB9hyN_
